src/cli/argument-parser.ts

```diff
@@ -126,6 +126,7 @@ export default class CLIArgumentParser {
             .option('--ts-config-path <path>', 'use a custom TypeScript configuration file and specify its location')
             .option('--cs, --client-scripts <paths>', 'inject scripts into tested pages', this._parseList, [])
             .option('--disable-page-caching', 'disable page caching during test execution')
+            .option('--screenshot-full-page', 'enable full page screenshots')
```

Comment:
Maybe we should go with plural `--screenshots-full-page`?

```
            .option('--screenshots-full-page', 'enable full-page screenshots')
```

Reply:
Not sure. But you are the one who decides =)
Please check .ts files

Reply:
oh, you mean a verb?
I though it's a namespace-like prefix...

well, in this case, yes, `--screenshot-full-page`

Reply:
Is it only this line in `.ts`? 
https://github.com/DevExpress/testcafe/pull/4265/files#diff-6baf1ad70587978c1199d08cc7f8298fR1196
It's fine

Reply:
We need description for methods:
` screenshots(options: ScreenshotsOptions): this;` - we have new overriding here
`takeScreenshot` - we have new parameter 'fullPage' here

Reply:
I would prefer the `--screenshots-full-page`, because I really wanted to introduce namespaces in our CLI arguments.